packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableRow.tsx

```diff
@@ -0,0 +1,189 @@
+          <ExternalLink
+            href={`${getExplorerUrl(tx.parentChainId)}/tx/${tx.txId}`}
+            className="mb-1"
+          >
+            <ArrowTopRightOnSquareIcon height={10} />
+          </ExternalLink>
+        </div>
+      )
+    }
+
+    // Success
+    return (
+      <div className="flex items-center space-x-1">
+        <CheckCircleIcon height={14} className="mr-1" />
+        <span>Success</span>
+        <ExternalLink
+          href={`${getExplorerUrl(tx.parentChainId)}/tx/${tx.txId}`}
+          className="mb-1"
+        >
+          <ArrowTopRightOnSquareIcon height={10} />
+        </ExternalLink>
+      </div>
+    )
+  }, [tx])
+
+  const isError = useMemo(() => {
+    if (tx.isCctp || !tx.isWithdrawal) {
+      if (
+        tx.depositStatus === DepositStatus.L1_FAILURE ||
+        tx.depositStatus === DepositStatus.EXPIRED
+      ) {
+        return true
+      }
+
+      if (tx.depositStatus === DepositStatus.CREATION_FAILED) {
+        // In case of a retryable ticket creation failure, mark only the token deposits as errors
+        return tx.assetType === AssetType.ETH
+      }
+    }
+
+    // Withdrawal
+    return tx.status === 'Failure'
+  }, [tx])
+
+  return (
+    <div
+      data-testid={`${isClaimableTx ? 'claimable' : 'deposit'}-row-${tx.txId}`}
+      className={twMerge(
+        'relative mx-4 grid h-[60px] grid-cols-[140px_140px_140px_140px_100px_180px_140px] items-center justify-between border-b border-white/30 text-xs text-white'
+      )}
+    >
+      <div className="pr-3 align-middle">{txRelativeTime}</div>
+      <div className="pr-3 align-middle">
+        {formatAmount(Number(tx.value), {
+          symbol: tokenSymbol
+        })}
+      </div>
+      <div className="flex items-center space-x-2">
+        <span>
+          <NetworkImage
+            chainId={tx.isWithdrawal ? tx.childChainId : tx.parentChainId}
+          />
+        </span>
+        <span className="inline-block w-[50px] break-words">
+          {getNetworkName(tx.isWithdrawal ? tx.childChainId : tx.parentChainId)}
+        </span>
+      </div>
+      <div className="flex items-center space-x-2">
+        <span>
+          <NetworkImage
+            chainId={tx.isWithdrawal ? tx.parentChainId : tx.childChainId}
+          />
+        </span>
+        <span className="inline-block w-[50px] break-words">
+          {getNetworkName(tx.isWithdrawal ? tx.parentChainId : tx.childChainId)}
+        </span>
+      </div>
+      <div className="pr-3 align-middle">
+        <StatusLabel />
+      </div>
+      <div className="flex justify-center pr-3 align-middle">
+        <TransactionsTableRowAction
+          tx={tx}
+          isError={isError}
+          type={tx.isCctp || !tx.isWithdrawal ? 'deposits' : 'withdrawals'}
+        />
+      </div>
+      <div className="align-middle">
+        <Button
+          variant="primary"
+          className="border border-white p-2 text-xs text-white"
+        >
```

Comment:
<img width="523" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/b76ddaa2-a904-4ed8-8caa-f4531284a7bc">
<img width="255" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/d30c053e-4195-4c79-9aa7-0fc4296eed2e">

the border radius should be 5px instead of 8px
same for other buttons

Reply:
Updated